package.json

```diff
@@ -8,7 +8,7 @@
   ],
   "scripts": {
     "build": "rimraf lib && babel src --out-dir lib",
-    "prepare": "npm run lint && npm run flow && npm run test && npm run build",
+    "prepare": "npm run build",
     "coveralls": "cat ./reports/lcov.info | coveralls",
```

Comment:
I didn't remove it my other PR, but this script and the NPM dependency can probably be dropped

Reply:
Ya, I was thinking the same thing.